### ISSUE TITLE
Warning after starting training: optimizer.step() called after lr-scheduler init?

### ISSUE THREAD

**Issue by Betm4n** [ultralytics/yolov5#392]
## ❔Question
Dear all, directly after starting the training, I get the following warning:

```
/usr/local/lib/python3.6/dist-packages/torch/optim/lr_scheduler.py:114: UserWarning: Seems like `optimizer.step()` has been overridden after learning rate scheduler initialization. Please, make sure to call `optimizer.step()` before `lr_scheduler.step()`. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
```

This is 100% reproducible with any configs i tried for every custom dataset (and anyway should be independed of the data set, right?
I just cloned the repo today, so it's latest/greatest.

train command:
!python train.py --img 1024 --batch 8 --epochs 2 --data /content/gdrive/"My Drive"/xxx.yaml --cfg ../content/gdrive/"My Drive"/yolov5x.yaml --name yolov5x_xxx  --weights ../content/gdrive/"My Drive"/yolov5x.pt 

I would highly appreciate any help.

**Comment by Laughing-q** [comment]
@Betm4n I think this issue might help you. #313

**Comment by Betm4n** [comment]
Thanks a lot, this solved it!
